compose/install.md

```diff
@@ -4,37 +4,52 @@ keywords: compose, orchestration, install, installation, docker, documentation
 title: Install Docker Compose
 ---
 
-You can run Compose on macOS, Windows and 64-bit Linux. To install it, you'll need to install Docker first.
+You can run Compose on macOS, Windows and 64-bit Linux.
 
-To install Compose, do the following:
+## Prerequisites
 
-1.  Install Docker Engine:
+You need to install Docker first, before you install and run Docker Compose.
```

Comment:
Technically that is not true - you can install Compose without having Docker installed first. You'll need access to a Docker engine to do anything useful with Compose, but that engine could be installed on a remote server.